File: Programmers/Lv.1/시저 암호/12926.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        
        String[] arr = s.split("");
        for (int i=0; i<s.length(); i++) {
            if (!" ".equals(arr[i])) {
                int c = s.charAt(i);
                if ((c <= 90 && (c + n) > 90) || (c + n) > 122) {
                    c -= 26;
                }
                
                arr[i] = "" + (char) (c + n);
            }
        }
        
        answer = String.join("", arr);
        
        return answer;
    }
}

File: Programmers/Lv.1/시저 암호/12926.js
function solution(s, n) {
    let answer = '';
    
    const arr = s.split('');
    for (let i=0; i<s.length; i++) {
        if (arr[i] !== ' ') {
            let c = s.charCodeAt(i);
            if ((c <= 90 && (c + n) > 90) || (c + n) > 122) {
                c -= 26;
            }
            
            arr[i] = String.fromCharCode(c + n);
        }
    }
    
    answer = arr.join('');
    
    return answer;
}
